source/HailPanel.cpp

```diff
@@ -214,14 +214,12 @@ void HailPanel::Draw()
 
 	DrawList draw;
 	// If this is a ship, copy its swizzle, animation settings, etc.
+	// Also draw its fighters and hardpoints.
```

Comment:
```suggestion
	// Also draw its fighters and weapon hardpoints.
```
May be better to be super specific here? Engine hardpoints do exist, even if it doesn't currently make sense to draw anything for them.